src/main/pages/che-7/administration-guide/proc_workspaces-requirements.adoc

```diff
@@ -95,11 +96,41 @@ For example, on a local Minikube cluster, the URL for the `che-incubator/typescr
 
 ... Custom CPU limit and request:
 +
-{prod-short} does not set CPU limits and requests. The `LimitRange` Kubernetes namespace may specify some defaults for CPU limits and requests. Kubernetes apply them to the `chePlugin` container.
+{prod-short} does not set CPU limits and requests by default. But it is still possible to configure CPU limits
+for `chePlugin` and `cheEditor` types via `meta.yaml` or devfile in the same way as it done for memory limits.
++
+.The `chePlugin` `che-incubator/typescript/latest`
+====
+.`meta.yaml` spec section:
+[source,yaml]
+----
+spec:
+ containers:
+   - image: docker.io/eclipse/che-remote-plugin-node:next
+     name: vscode-typescript
+     cpuLimit: 2000m
+     cpuRequest: 500m
+----
+
+It results in a container with the following CPU limit and request:
+
+|===
+|CPU limit | 2 cores
+|CPU request | 0.5 cores
+|===
+====
+
+To set CPU limits and request globally, there is dedicated environment variables in {prod-short}.
+For `chePlugin` and `cheEditor` types, those are `+CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__LIMIT__CORES+`
+and `+CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__REQUEST__CORES+`. Please refer to variables description page for more information.
+
+Also, need to keep in mind that the `LimitRange` object of Kubernetes namespace may specify some defaults for CPU limits and requests set by cluster administrators, so setting limits on application or workspace levels must comply with those settings,
+otherwise it is possible to get workspace startup errors due to resources overrun.
+
 
 .. Custom requirements for `dockerimage` components
 +
-If present, the `memoryLimit` attribute of the devfile defines the memory limit of a `dockerimage` container. {prod-short} set automatically the memory request to match the memory limit. There is no attribute in a devfile to set the memory request.
+If present, the `memoryLimit` and `memoryRequest` attribute of the devfile defines the memory limit of a `dockerimage` container. {prod-short} sets automatically the memory request to match the memory limit in case it is not specified explicitly.
```

Comment:
```suggestion
If present, the `memoryLimit` and `memoryRequest` attributes of the devfile define the memory limit of a `dockerimage` container. {prod-short} automatically sets the memory request to match the memory limit in case it is not specified explicitly.
```